cobra/flux_analysis/find_active_reactions.py

```diff
@@ -0,0 +1,583 @@
+
+"""
+Find all active reactions by solving a single MILP problem
+or a (small) number of LP problems
+
+"""
+from cobra.flux_analysis.variability import flux_variability_analysis
+from cobra.flux_analysis.helpers import normalize_cutoff
+from optlang import Model, Variable, Constraint, Objective
+from optlang.symbolics import Zero
+from scipy.linalg import orth
+import numpy as np
+
+
+def relax_model_bounds(model, bigM=1e4):
+    """
+    Relax all upper and lower bounds in the model.
+    All positive upper bounds will become bigM.
+    All negative lower bounds will become -bigM.
+    All positive lower bounds and negative upper bounds will become zero.
+
+    Parameters
+    ----------
+    model: cobra.Model
+        cobra model. It will *not* be modified.
+    bigM: float, optional
+        a large constant for relaxing the model bounds, default 1e4.
+
+    Returns
+    -------
+    cobra.Model
+        cobra model with relaxed bounds
+
+    """
+
+    for r in model.reactions:
+        r.upper_bound = bigM if r.upper_bound > 0 else 0
+        r.lower_bound = -bigM if r.lower_bound < 0 else 0
+
+
+def find_active_reactions(model, bigM=10000, zero_cutoff=None,
+                          relax_bounds=True, solve="lp", verbose=False):
+    """
```

Comment:
Rather than using a verbose flag and print, use the logging module as done in other modules and output the additional messages with INFO or DEBUG. That allows user to set the verbosity globally rather than individually for each function.